### PR TITLE
Remove TestDeleteBlob UT

### PR DESCRIPTION
**Pull request by lzhecheng**
Signed-off-by: Zhecheng Li <zhechengli@microsoft.com>

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/azure/azure_util_test.go [hunk 1]
```diff
@@ -23,21 +23,16 @@ import (
...
 	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
 )
 
 const (
-	testAccountName = "account"
-	// TODO(nilo19): verify if DeleteBlob unit test needs to check error string, and which message is expected. We saw both:
-	// storageAccountClientErrMsg = "Server failed to authenticate the request. Make sure the value of Authorization " +
-	//	 "header is formed correctly including the signature"
-	// storageAccountClientErrMsg = "The specified account is disabled"
+// TODO(nilo19): verify if DeleteBlob unit test needs to check error string, and which message is expected. We saw both:
```
Comment (marwanad) [inline cluster-autoscaler/cloudprovider/azure/azure_util_test.go:32]: should we reove this comment as well?

Comment (lzhecheng) [reply cluster-autoscaler/cloudprovider/azure/azure_util_test.go]: I think so. Thanks.

**Comment by marwanad** [comment]
@lzhecheng can we cherry-pick this backwards to 1.23, 1.22 and 1.21 branches to unblock people making cherry-picks?

**Comment by lzhecheng** [comment]
> @lzhecheng can we cherry-pick this backwards to 1.23, 1.22 and 1.21 branches to unblock people making cherry-picks?

Thanks for reminding! I am going to do it now.

**Pull request merged**